packages/framework/oldest-client-observer/package.json

```diff
@@ -86,8 +87,9 @@
     "typescript": "~4.5.5"
   },
   "typeValidation": {
-    "disabled": true,
```

Comment:
Since oldest-client-observer is still experimental, I'd leave these disabled or I assume we'll hit the same issue when we go to re-publish as non-experimental.